components/button/button.css

```diff
@@ -0,0 +1,111 @@
+.btn-block {
+  min-width: 100%;
+  max-width: 100%;
+}
+
+.btn-link {
+  padding: 0;
+  background-color: transparent;
+}
+
+.btn-disabled {
+  cursor: not-allowed;
+}
+
+.btn-lg {
+  padding: 0.8rem 1.6rem;
+  font-size: 1.5rem;
+  line-height: 2rem;
+}
+
+.btn-md {
+  padding: 0.6rem 1.2rem;
+  font-size: 1rem;
+  line-height: 1.5rem;
+}
+
+.border-pill {
+  border-radius: 2rem;
+}
+
+.border-round {
+  border-radius: 50%;
+}
+
+.btn-sm {
+  padding: 0.4rem 0.8rem;
+  font-size: 0.8rem;
+  line-height: 1.2rem;
+}
+
+.btn-disabled-primary {
+  border: 1px solid var(--dark);
+}
+
+.btn-flote-primary {
+  box-shadow: -1px -1px 10px 5px rgb(161 177 188);
```

Comment:
you can use variable for this color too.

Reply:
I did, thanks for reviewing